### ISSUE TITLE
Install nvQuickSite on Windows 11

### ISSUE THREAD

**Issue by Senyors** [nvisionative/nvQuickSite#351]
I just purchased a new Dell Inspiron running Windows 11 and I need to set up a DNN instance. Running nvQuickSite, I soon get the following error:

![image](https://user-images.githubusercontent.com/6136932/146296001-5fb43732-240e-4421-b15a-ab7c7eaef54b.png)

Is Windows 11 not supported? Am I perhaps doing something wrong?

**Comment by david-poindexter** [comment]
@Senyors yes, nvQuickSite works great on Windows 11 - just be sure you meet [the minimum requirements](https://www.nvquicksite.com/Requirements).

**Comment by Senyors** [comment]
Thanks, David. I have VS 2022 installed along with SQL Server 2019, but I didn't have IIS installed. After doing that, I'm now getting a permissions issue...

![image](https://user-images.githubusercontent.com/6136932/146298522-77470485-b918-4568-b550-a62250a0f2e4.png)

Any ideas what that may be? It seems strange that it wants a path to the database.

I've used nvQuickSite dozens of times in the past on older machines but this is my first try with Windows 11. I also tried to look up the DNN install process online (https://docs.dnncommunity.org/content/getting-started/setup/installation/index.html) but there's not much there.

**Comment by david-poindexter** [comment]
@Senyors I would recommend trying to install outside of your user directory to avoid potential permissions issues due to the "specialness" of those folders.

You can also enable verbose logging in nvQuickSite settings to look closer at the logs for clues.  

Are you running the latest version of nvQuickSite?

Also, what are you using for your database server name?  If you are using `(local)` this can sometimes be problematic.  Instead, try the DB instance name.

**Comment by Senyors** [comment]
Your last suggestion worked for me--I had to enter the actual name of the database instance. DNN installed, but when trying to run the new site, I get this error:

![image](https://user-images.githubusercontent.com/6136932/146300333-248220ab-b2e3-46b7-9aff-04ecc0635c25.png)

**Comment by david-poindexter** [comment]
I'd recommend installing outside your "Users" folder to avoid a plethora of issues that can occur there.

Also, you want to make sure IIS is properly configured.  [This article](https://www.dnnsoftware.com/community-blog/cid/155489/iis-gotchas-when-installing-dnn) may help.

**Comment by Senyors** [comment]
Thanks for all the help, David. The article was required reading; after checking all the boxes featured in the article on my own PC, nvQuickSite ran perfectly and the site is now up and running. I'll close this issue, as it was all my own ignorance with Windows 11.

**Comment by david-poindexter** [comment]
Awesome - so glad you are up and running now - welcome to Windows 11. 😊👍🏼

**Comment by Senyors** [comment]
Thank you sir! And what an honor to chat with a legendary person in the DNN realm--thank you again for the quick and extremely helpful response!

**Comment by david-poindexter** [comment]
@Senyors I can assure you the pleasure is all mine sir!  This DNN Community rocks!

**Comment by tsc646167645** [comment]
@Senyors I get  the same error during installing  nvQuickSite with  Windows 11. 
![image](https://github.com/user-attachments/assets/c21e28cd-2f36-47ce-9d61-4ad7829d59de)
SIr, Can you describe how you solved this problem? This is my first time using DNN.

**Comment by valadas** [comment]
Seaching for that GUID, it belongs to IIS. Did you install IIS, is it working fine ?

**Comment by david-poindexter** [comment]
@ShiChunTang see the article referenced in [the comment above](https://github.com/nvisionative/nvQuickSite/issues/351#issuecomment-995396801).  This will help to ensure IIS is configured properly.

**Comment by ShiChunTang** [comment]
> Seaching for that GUID, it belongs to IIS. Did you install IIS, is it working fine ?

@valadas  @david-poindexter  I got a bug about installing .net framework 3.5 in my Windows11 23H2 Insider Preview (10.0.22635.3930) and after reading the Microsoft blog I confirmed that this bug was inseparable from my personal fragile Wndows11. I had to install a new Windows 11 23H2 to fix this serious issue. After that, the DNN became available, and I don't think it's reproducible. I am a young self-learner with little development experience, thank you very much for your help!
